src/core/ChainRequest.js

```diff
@@ -0,0 +1,56 @@
+import fetch from 'node-fetch';
```

Comment:
when a file is named SomethingCamelCaseStartingWithCapitalLetter, by convention it means that its default export is a class by that name. since this is not the case here, it should be named something else. looking at the contents of this file, it seems like a good name for it would be `api.js`

Reply:
Good to know!